tasks/publish.sh

```diff
@@ -28,6 +28,7 @@ root_path=$PWD
 
 if [ -n "$(git status --porcelain)" ]; then
   echo "Your git status is not clean. Aborting.";
+  git diff
```

Comment:
I think maybe `git status` would be more appropriate here. `git diff` could end up printing a lot if you have lots of changes.